src/pcapi/routes/pro/bookings.py

```diff
@@ -101,11 +104,29 @@ def get_all_bookings(query: ListBookingsQueryModel) -> ListBookingsResponseModel
 
 
 # @debt api-migration
-@public_api.route("/v2/bookings/token/<token>", methods=["GET"])
+@pro_api_v2.route("/bookings/token/<token>", methods=["GET"])
 @ip_rate_limiter(deduct_when=lambda response: response.status_code != 200)
 @email_rate_limiter(key_func=get_basic_auth_from_request, deduct_when=lambda response: response.status_code != 200)
+@spectree_serialize(
+    api=api,
+    response_model=GetBookingReponse,
+    tags=["API Contremarque"],
+    on_error_statuses=[401],
+    code_descriptions={
+        "HTTP_200": "La contremarque existe et n’est pas validée",
+        "HTTP_401": "Authentification nécessaire",
+        "HTTP_403": "Vous n'avez pas les droits nécessaires pour voir cette contremarque",
+        "HTTP_404": "La contremarque n'existe pas",
+        "HTTP_410": "La contremarque n'est plus valide car elle a déjà été validée ou a été annulée",
+    },
+)
 @login_or_api_key_required
-def get_booking_by_token_v2(token: str):
+def get_booking_by_token_v2(token: str) -> GetBookingReponse:
+    """Consultation d'une réservation.
+
+    Le code “contremarque” ou "token" est une chaîne de caractères permettant d’identifier la réservation et qui sert de preuve de réservation.
+    Ce code unique est généré pour chaque réservation d'un utilisateur sur l'application et lui est transmis à cette occasion.
+    """
```

Comment:
Cette docstring est en français pour être affichée dans la doc générée par Swagger ?

(Si oui, ça me semble raisonnable. Éventuellement on pourra ajouter un commentaire Python pour le préciser. :

```suggestion
    """  # in French, to be used by Swagger for the API documentation
```

Reply:
oui c'est ça. ok. j'ai pas trop moyen de mettre le commentaire au niveau des guillemets par contre
![image](https://user-images.githubusercontent.com/22373097/131984713-209fcd5b-5a88-49a4-8eae-47e459385162.png)